probes/codeApproved/def.yml

```diff
@@ -0,0 +1,21 @@
+# © 2023 Nokia
+# Licensed under the Apache License 2.0
+# SPDX-License-Identifier: Apache-2.0

```

Comment:
Thanks for changing this to Apache 2.0

Can we keep the license text similar to what we have in the other files?

```
// Copyright 2023 OpenSSF Scorecard Authors
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//      http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
```

Reply:
Note: I did ask about listing `Nokia` and was told that it's a valid option. 

```
// Copyright 2023 Nokia
```

When someone else edits the file we will need to change it to include other authors as well. Which is partly why we use "OpenSSF Scorecard Authors" now as a catchall, so we don't need to go and update it the next time someone updates this file. Nokia would still own the copyright of your contribution.

Reply:
I checked with my open source specialist, and it's ok, I can change it to the suggested license text.